idl/github.com/uber/cadence/history.thrift

```diff
@@ -318,8 +319,25 @@ service HistoryService {
       4: ShardOwnershipLostError shardOwnershipLostError,
       5: shared.LimitExceededError limitExceededError,
       6: shared.ServiceBusyError serviceBusyError,
+      7: shared.CurrentBranchChangedError currentBranchChangedError,
     )
 
+  /**
+   * Returns the information from mutable state of workflow execution.
+   * It fails with 'EntityNotExistError' if specified workflow execution in unknown to the service.
+   * It returns CurrentBranchChangedError if the workflow version branch has changed.
+   **/
+   GetMutableStateResponse GetMutableStateWithLongPoll(1: GetMutableStateRequest getRequest)
```

Comment:
input and output should be a different type

Reply:
separated the request and response